content/applications/sales/crm/pipeline/lost_opportunities.rst

```diff
@@ -3,51 +3,51 @@ Manage lost opportunities
 =========================
 
 Not every opportunity results in a successful sale. To keep the pipeline up-to-date, *lost*
-opportunities need to be identified. Specifying why an opportunity was lost provides additional
-insight that can prove useful for future opportunities.
+opportunities need to be identified. Specifying why an opportunity was lost helps identify recurring
+issues, reveal coaching opportunities, and can assist with improving overall sales strategy.
+
+.. note::
+   :doc:`Merging lost opportunities <merge_similar>` with active ones will pull them back into the
+    pipeline.
 
 Mark an opportunity as lost
 ===========================
 
-To mark an opportunity as lost, first open the :menuselection:`CRM app`, and select an opportunity
-from the pipeline, by clicking on its corresponding Kanban card. Doing so reveals that opportunity's
+To mark an opportunity as lost, first open the :menuselection:`CRM` app, then select an opportunity
+from the pipeline by selecting its corresponding Kanban card. This reveals that opportunity's
 detail form.
 
-Then, click :guilabel:`Lost`, located at the top of the opportunity's detail form.
+Then select :guilabel:`Lost`, located at the top of the form.
 
 .. image:: lost_opportunities/lost-opps-lost-button.png
-   :align: center
    :alt: Buttons from the top of an opportunity record with the lost button emphasized.
 
 This opens the :guilabel:`Mark Lost` pop-up window. From the :guilabel:`Lost Reason` drop-down menu,
-choose an existing lost reason. If no applicable reason is available, create a new one by entering
-it into the :guilabel:`Lost Reason` field, and clicking :guilabel:`Create`.
+choose an existing lost reason. If none of these are appropriate, create a new one by entering it
+into the :guilabel:`Lost Reason` field, then select :guilabel:`Create`.
```

Comment:
Pretty sure this sentence and others are missing "and" conjunctions? 
"If" statements ideally follow with a quick "then" consequence.
Another example of a sentence that did not need to be rewritten.

```suggestion
choose an existing lost reason. If none of these are appropriate, then create a new one by entering it
into the :guilabel:`Lost Reason` field, and then clicking :guilabel:`Create`.
```